test/unit/process_while_test.lua

```diff
@@ -0,0 +1,47 @@
+local expirationd = require('expirationd')
+local t = require('luatest')
+local g = t.group("process_while")
+
+local helpers = require('luatest.helpers')
+
+g.before_all(function()
+    helpers.init_spaces(g)
+end)
+
+g.after_each(function()
+    helpers.truncate_spaces(g)
+end)
+
+function g.test_stopper()
+    helpers.iteration_result = {}
+    local function process_while(task)
+        if task.checked_tuples_count >= 1 then return false end
+        return true
+    end
+
+    -- for tree index spaces
+    for _, space in pairs({g.tree, g.vinyl}) do
```

Comment:
nit: I see you try to invent some kind of parametrisation for test. Try to use example from crud - https://github.com/tarantool/crud/blob/master/test/pgroup.lua .

Reply:
since this is a nit, I decided to leave it as it is, since it is important to close this task faster